app.rb

```diff
@@ -0,0 +1,25 @@
+
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  get '/' do
+    erb :homepage
+  end
+
+  get '/feed' do
+    @peeps = Peep.all
+    erb :index
+  end
+
+  get '/feed/create' do
```

Comment:
the RESTful way to name this route would be something like `get '/peep/create'`.

the format for the URI is '/resource/action', and what you're doing here is creating a new peep, not a new feed. so it should be '/peep/create'.